storm-core/src/clj/org/apache/storm/config.clj

```diff
@@ -91,25 +91,25 @@
   (let [storm-home (System/getProperty "storm.home")
         path (conf STORM-LOCAL-DIR)]
     (if path
-      (if (is-absolute-path? path) path (str storm-home file-path-separator path))
-      (str storm-home file-path-separator "storm-local"))))
+      (if (is-absolute-path? path) path (str storm-home Utils/filePathSeparator path))
+      (str storm-home Utils/filePathSeparator "storm-local"))))
```

Comment:
The pull request for Config.clj #1030 and the pull request for zookeeper #1045 each ported the path separator differently.  Can we at least file a JIRA to make it common in the future.